src/main/archetype/all/pom.xml

```diff
@@ -0,0 +1,196 @@
+ |  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- ====================================================================== -->
+    <!-- P A R E N T  P R O J E C T  D E S C R I P T I O N                      -->
+    <!-- ====================================================================== -->
+    <parent>
+        <groupId>${groupId}</groupId>
+        <artifactId>${rootArtifactId}</artifactId>
+        <version>${version}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T  D E S C R I P T I O N                                   -->
+    <!-- ====================================================================== -->
+    <artifactId>${artifactId}</artifactId>
+    <packaging>content-package</packaging>
+    <name>${artifactName} - All</name>
+    <description>All content package for ${artifactName}</description>
+
+    <!-- ====================================================================== -->
+    <!-- B U I L D   D E F I N I T I O N                                        -->
+    <!-- ====================================================================== -->
+    <build>
+        <plugins>
+            <!-- ====================================================================== -->
+            <!-- V A U L T   P A C K A G E   P L U G I N S                              -->
+            <!-- ====================================================================== -->
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <group>${packageGroup}</group>
+                    <embeddeds>
+                        <embedded>
+                            <groupId>${groupId}</groupId>
+                            <artifactId>${rootArtifactId}.ui.apps</artifactId>
+                            <type>zip</type>
+                            <target>/apps/${appsFolderName}-packages/application/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>${groupId}</groupId>
+                            <artifactId>${rootArtifactId}.ui.content</artifactId>
+                            <type>zip</type>
+                            <target>/apps/${appsFolderName}-packages/content/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.cq</groupId>
+                            <artifactId>core.wcm.components.all</artifactId>
```

Comment:
I believe the Core Components dependencies should be targetted separately into their respective `/apps/vendor-packages/application/...` and `/apps/vendor-packages/content/...` folders. Core Components All pacakges itself is a mixed package right?

Reply:
@vladbailescu -  @davidjgonzalez  is correct - the all wrappers should optimally not be used directly. The "challenge" of core components is that the core bundle is embedded in the "all" package instead of the application package (cc @bpauli): As this is our blueprint I would suggest to adapt the nesting structure of the core components. You "could" also embedd the core bundle directly but I fear that makes things more confusing as the container would no longer be a pure container. Embedding the core components bundle into the projects application package might also be viable but is also just a workaround for the core components having this bundle currently in a MIXED package (sth I already discussed with @bpauli  to be fixed)

Reply:
@vladbailescu , @bpauli , @davidjgonzalez  - thinking about it twice this sounds not too wrong- although I would suggest to name it /apps/${appsFolderName}-vendorpackages -  which may contain other "all" packages of projects but will prevent collisions when multiple partners are acting on multiple projects/applications that would be combined/integrated into one environment - The semantics would only allow be for nesting libraries where you would want to nest the containers (such as core components all, or acs commons) - dependency handling would be orthogonal and should never go against those all package ids.

Reply:
Wouldn't `ui.content` and `ui.apps` be enough to embed?

Reply:
Done in https://github.com/adobe/aem-project-archetype/pull/256/commits/b4b94d13dfd5506e5e2d5d617b41fa3de66fb10a#diff-9cfeab39e66fba55ca8235d10aedb5adR68-R74